src/lib/stable/ic_apis/call.ts

```diff
@@ -42,7 +42,9 @@ export async function call<Args extends any[] | undefined, Return = any>(
             } else {
                 const idlType =
                     returnTypeIdl === undefined ? [] : [returnTypeIdl];
-                resolve(idlDecode(idlType, result)[0] as Return);
+                resolve(
```

Comment:
Change result type to be just Uint8Array, and get rid of new Uint8Array

Reply:
that will break for experimental with stable api